addOns/webdrivers/webdrivers.gradle.kts

```diff
@@ -31,20 +31,43 @@ fun configureDownloadTask(outputDir: File, targetOs: DownloadWebDriver.OS, task:
     }
 }
 
+fun getArchPath(task: DownloadWebDriver): String {
+    return when (task.arch.get()) {
+        DownloadWebDriver.Arch.X32 -> "32"
+        DownloadWebDriver.Arch.ARM64 -> "ARM64"
+        else -> "64"
+    }
+}
+
 subprojects {
+
+    val webdriver = configurations.create("webdriver") {
```

Comment:
`val webdriver by configurations.creating {` the name could be `wdm` or `selenium`.

Reply:
This one is still pending, better use the`by` construct which does not require duplicate the name of the configuration.